pep-9999.rst

```diff
@@ -47,13 +47,47 @@ required to interpret all standard constructs, and additionally a type checker
 can support extra constructs not described in this PEP. All deviations from
 this PEP should be documented.
 
+Syntax
+======
+
+Type stubs are syntactically valid Python files with a ``.pyi`` suffix.
+They should be valid according to the latest Python version six months
+after release at the latest, but can start to use features from that
+release from the release date on.
+Type stubs authors are encouraged to use the latest available syntax
+features in stubs, even if the type information in the stub is targeted
+to an older Python version.
+
+Type checkers are expected to support the syntax of the latest released Python
+version.
+
+For example, Python 3.7, which reserved keywords `async` and `await`
+as keywords (see [#pep492]_), was released on June 27nd, 2018. Type
+checkers should support them by that date and type stubs could start
+using `async` to mark function definitions. Additionally, type stubs
+should stop using the new keyword as an attribute or function name before
+December 27nd, 2018.
+
+Distribution
+============
+
+Type stubs can be distributed separately from the implementation;
```

Comment:
PEP 561 also formalizes how to distribute type information *with* the implementation.
```suggestion
Type stubs can be distributed with or separately from the implementation;
```